CHANGELOG.md

```diff
@@ -1,3 +1,145 @@
+# 0.32.0 (2019-??-??)
+
+## Language changes
+
+- Allow boolean negation to be written also as a regular method call `expr.!`. ([#8445](https://github.com/crystal-lang/crystal/pull/8445), thanks @jan-zajic)
+- Support named arguments in macro methods. ([#8429](https://github.com/crystal-lang/crystal/pull/8429), thanks @Blacksmoke16)
```

Comment:
That's confusing. Named arguments have been supported in user-space macro methods before. #8429 just enables evaluating named arguments in the macro interpreter. That's not a language change, but a compiler feature adding general support + implementation of `StringLiteral#camelcase(*, lower)`.

Reply:
In fact named arguments are just supported for one particular method. So maybe that should be the changelog. It doesn't work if you use named arguments in general for other macro methods

Reply:
Yeah, it's only support for named-only parameters.

Reply:
And only for the camel case method.

Eventually I'd like the macro methods to be registered similar to how regular methods are. Then when trying to find a macro method we would use a similar logic for method lookup. We would have to hardcode each method's logic for checking args. But I don't know how easy that is because of the dynamic nature of macro methods.